### DISCUSSION TITLE
Youtube saying "Please use a keyframe frequency of four seconds or less."

### DISCUSSION THREAD

**Discussion by gulungse** [ant-media/Ant-Media-Server#5679] [Q&A]
Hello

OK so I watched a popular YouTube video of how to create a 24/7 YouTube Livestream (link below) where he used Linode to connect to Ant Media Server.

https://www.youtube.com/watch?v=0scjy6Zxzwcwed

The issue I'm having is that YouTube says I have poor stream health - see message below:

"Please use a keyframe frequency of four seconds or less. Currently, keyframes are not being sent often enough, which can cause buffering. The current keyframe frequency is 5.8 seconds. Note that ingestion errors can cause incorrect GOP (group of pictures) sizes."

How can I configure this in Ant Media whilst connected to Linode?

Please let me know.

Thanks!

**Comment by USAMAWIZARD** [comment]
Hi
I hope you are doing well.

You can use ffmpeg to change the key frame interval. Below command will encode the file with key frame interval of 2.
 
ffmpeg -i input.mp4 -qscale 0 -g 2 outputFile.mp4

credits @yashtandon113  
answer was originally posted [here](https://community.antmedia.io/topic/1532-youtube-saying-please-use-a-keyframe-frequency-of-four-seconds-or-less/#comment-5593)
